proxies/live/apiproxy/policies/OAuthV2.GenerateAccessToken.xml

```diff
@@ -16,7 +16,7 @@
         <Attribute name="id_token" ref="externalIdToken" display="true"/>
         <Attribute name="id_token-subject" ref="jwt.DecodeJWT.FromExternalIdToken.claim.subject" display="true"/>
         <Attribute name="id_token-issuer" ref="jwt.DecodeJWT.FromExternalIdToken.claim.issuer" display="true"/>
-        <Attribute name="id_token-acr" ref="jwt.DecodeJWT.FromExternalIdToken.claim.acr" display="true"/>
```

Comment:
I don't think we want to replace the `id_token-acr` with `authentication_assurance_level` necessarily as if might cause backward compatibility issues. It should probably just be a new variable i.e. `id_token-authentication_assurance_level`. Once we've made the change it would be good to update the documentation too: https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?spaceKey=APM&title=Identity+service+token+variables

Reply:
Made the changes and Added new Attribute